content/en-us/reference/engine/enums/AdUnitStatus.yaml

```diff
@@ -1,18 +1,22 @@
 name: AdUnitStatus
 type: enum
-summary: ''
-description: ''
+summary: |
+  Exposes the status of an immersive ad.
+description: |
+  Exposes the status of an immersive ad.
 code_samples: []
 tags: []
 deprecation_message: ''
 items:
   - name: Inactive
     value: 0
-    summary: ''
+    summary: |
+      The ad unit isn't currently serving an ad. It may be configured incorrectly. An `Class.AdGui` with this status will display its fallback image.
```

Comment:
```suggestion
      The ad unit isn't currently serving an ad and it may be configured incorrectly. An `Class.AdGui` with this status will display its fallback image.
```